tools/jdbc/src/main/java/org/duckdb/DuckDBTimestamp.java

```diff
@@ -94,15 +94,15 @@ public String toString() {
 		return this.toLocalDateTime().toString();
 	}
 
-	private static long micros2seconds(long micros) {
+	public static long micros2seconds(long micros) {
```

Comment:
These shouldn't be public, as they're mostly implementation details - package private should suffice

Reply:
I was wondering if I should add other public `DuckDBTimestamp` methods that use them but all the good method names were taken...